aws-lambda/container/.serverless-wsgi

```diff
@@ -0,0 +1 @@
+{"app":"pygeoapi/flask_app.APP","text_mime_types":["application/ld+json"]}
```

Comment:
Do we need to add all the default supported text MIME types here?

- `application/json`
- `text/html`
- `application/geo+json`
- `application/prs.coverage+json`
- `application/xml`